src/nvim/option.c

```diff
@@ -7272,9 +7272,12 @@ int win_signcol_count(win_T *wp)
   // yes or yes
   if (!strncmp(scl, "yes:", 4)) {
     // Fixed amount of columns
-    return scl[4] - '0';
+    maximum = scl[4] - '0';
+    if (!wp->w_float_config.minimal) {
+      return maximum;
+    }
   }
-  if (*scl == 'y') {
+  if (*scl == 'y' && !wp->w_float_config.minimal) {
```

Comment:
'signcolumn' is window-local. `set_minimal_options` could set it to "auto" instead of special-casing here.

Reply:
I wanted to preserve the number X in `yes:X` to behave like `auto:X` without doing string editing. Though this would be less invasive in this case than the `fcs` case.